Keyboards/LanguageKeyboards/Hebrew/HECommandVariables.swift

```diff
@@ -10,35 +10,136 @@
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
 func heSetConjugationLabels() {
   // Reset all form labels prior to assignment.
-  for k in formLabelsDict.keys {
-    formLabelsDict[k] = ""
+  for key in formLabelsDict.keys {
+    formLabelsDict[key] = ""
+  }
+
+  switch heConjugationState {
+  case .present, .past, .future, .conditional:
+    formLabelsDict["TL"] = "פועל פשוט" // Poel Pashoot (Simple Verb)
+    formLabelsDict["TR"] = "פועל מתמשך" // Poel מתמשך (Continuous Verb)
+    formLabelsDict["BL"] = "פועל בתר" //  (Past Perfect Verb)
+    formLabelsDict["BR"] = "פועל בתר מתמשך" // Poel Be'עבר מתמשך (Past Continuous Perfect Verb)
+  case .presSimp, .presPerf, .presPerfCont:
+    formLabelsDict["Left"] = "אני/אתה/את/אתם" // Ani/Ata/At/Atem (I/You/They)
+    formLabelsDict["Right"] = "הוא/היא" // Hu/Hi (He/She)
+  case .presCont:
+    formLabelsDict["TL"] = "הווה משתתף" // Hove משתתף (Present Participle)
+    formLabelsDict["TR"] = "אני" // Ani (I)
+    formLabelsDict["BL"] = "אתה/את/אתם" // Ata/At/Atem (You/They)
+    formLabelsDict["BR"] = "הוא/היא" // Hoo/Hi (He/She)
+  case .pastCont:
+    formLabelsDict["Top"] = "היה משתתף" // Haya משתתף (Past Participle)
+    formLabelsDict["Middle"] = "אני/הוא/היא" // Ani/Hu/Hi (I/He/She)
+    formLabelsDict["Bottom"] = "אתם/אתן" // Atem/Aten (You [plural, masculine/feminine])
   }
 }
 
 /// What the conjugation state is for the conjugate feature.
 enum HEConjugationState {
   case present
+  case presSimp
```

Comment:
So here we'll need to update this list based on what conjugations we'll get from Wikidata :)